.github/ISSUE_TEMPLATE/20-performance-issue.md

```diff
@@ -0,0 +1,59 @@
+---
+name: Performance Issue
```

Comment:
This template is virtually identical to the bug one so we can remove it. Users can file performance issues as bugs.

Reply:
Sure.